submissions/dev-adventure/index.js

```diff
@@ -0,0 +1,77 @@
+const net = require('net');
+
+const startCheck = async () => {
+  try {
+    const arg = process.argv.slice(2);
+    if (!arg || !arg.length) {
+      process.stdout.write(
+        '--host <domain name or ip what you want to check>' +
+          ' is required arguments.\n' +
+          '--help for get description.'
+      );
+      process.exit(1);
+    }
+    if (arg[0] === '--help') {
+      process.stdout.write(
+        '--host <domain name or ip what you want to check> - required argument.\n' +
+          '--ports <start_port-end_port> - ports range from/to.\n' +
+          '--help - get help.'
+      );
+    }
+    let host;
+    let portStart;
+    let portEnd;
+    if (arg.length > 2) {
+      const ports = arg[1].split('-');
+      portStart = Number(ports[0]);
+      portEnd = Number(ports[1]);
+      [host] = arg.slice(3, 4);
```

Comment:
Isn't it the same as `const host = arg[3]`?

Reply:
We need to user host in outer block, but if would write `host . arg[3]` we get eslint error.

Reply:
Oh, you are right, linter enforces array destructuring. Here is what I suggest. 
1. First of all, do not rely on particular order of arguments to be passed to your program. Agreed, this was not specified in the task, but it is common to accept arguments in any order. So instead of taking the value at a particular index iterate over the array and take the next item after parameter name.
2. Create a separate function which will parse the arguments and returns them as an object. This way you will avoid having mutable variables and also you will abstract arguments parsing away.
I imagine the following interface:
```
const args = const arg = process.argv.slice(2);
const { host, startPort, endPort } = parseArgs(args);

```

Reply:
Good! Thanks :)